docs/installation.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Getting Started
+order: 1
+---
```

Comment:
This change moves the installation step as the first one in the nav.  Otherwise the Toucan Form stuff would be at the top